tls/s2n_connection.h

```diff
@@ -306,7 +306,7 @@ struct s2n_connection {
     uint32_t ticket_lifetime_hint;
 
     /* Session ticket extension from client to attempt to decrypt as the server. */
-    uint8_t ticket_ext_data[S2N_TLS12_TICKET_SIZE_IN_BYTES];
+    uint8_t ticket_ext_data[S2N_MAX_TICKET_SIZE_IN_BYTES];
     struct s2n_stuffer client_ticket_to_decrypt;
```

Comment:
How much bigger is S2N_MAX_TICKET_SIZE_IN_BYTES? Should you switch to heap memory?

Reply:
Just realized I do not need to resize the stuffer to do this pr. It makes more sense for that to be in the next pr, where we are actually calling the decrypt method while reading psks.